### ISSUE TITLE
duplicated slug for csc event

### ISSUE THREAD

**Issue by evykassirer** [wics-uw/website#238]
```
WARNING: There are 2 variants of "wics-csc-s16" with lang en
WARNING:    /tmp/build_fc0927ffb54766d5c964bdd3120814e8/wics-uw-website-47f9fdf/content/S2016/S16-updated-event-wics-csc.md
WARNING:    /tmp/build_fc0927ffb54766d5c964bdd3120814e8/wics-uw-website-47f9fdf/content/S2016/S16-event-wics-csc.md
```

Sorry Bhargavi, I missed this in code review. Can you fix it please?

**Comment by evykassirer** [comment]
Fatema hot fixed this in c703dfd

**Comment by evykassirer** [comment]
(got confused and didn't realize that Bharg's pr was doing that exact thing)
